etc/packaging/appimages/cartographer-module-aarch64.yml

```diff
@@ -77,6 +77,7 @@ AppDir:
     - libglx0:arm64
     - libgl1:arm64
     - libglvnd0:arm64
+    - libnlopt0:arm64
```

Comment:
[note to self] We have our own brew formula for nlopt0 that's [pinned](https://github.com/viamrobotics/homebrew-brews/blob/35e903c5be3f50fc7a1ec294c40302cd5d14e45d/Formula/nlopt-static.rb#L4) at v2.7.1 so that installations through brew will use that specific version.

This should also include v2.7.1, since that's the version of libnlopt0 in bullseye: https://packages.debian.org/bookworm/libnlopt0